Formula/gaze.rb

```diff
@@ -0,0 +1,37 @@
+class Gaze < Formula
+  desc "Execute commands for you"
+  homepage "https://github.com/wtetsu/gaze"
+  url "https://github.com/wtetsu/gaze/archive/refs/tags/v1.1.1.tar.gz"
+  sha256 "02bac4bfeded4e8e951af2d255a26ea3554dbbf4fd124cc9cfad5e29265e0a4d"
+  license "MIT"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", "-o", "dist/gaze", "-v", "cmd/gaze/main.go"
+    bin.install "dist/gaze"
+  end
+
+  test do
+    (testpath/"test.txt").write("init")
+    o = IO.popen("#{bin}/gaze -c 'cp test.txt out.txt' test.txt")
+    thread = Thread.new do
+      10.times do
+        begin
+          File.write("test.txt", "hello, world!")
+        rescue
+          # Safe to ignore
+        end
+        sleep 1
+      end
+    end
+    10.times do
+      break if File.exist?("out.txt")
+
+      sleep 1
+    end
+    thread.terminate
```

Comment:
Why repeat this 10 times?

Reply:
Thank you for your review.

* L16: A process starts
* L17~26: Write a file (test.txt)
* L27~31: Wait for file (out.txt)

The process watches the file system.
If the process detected a file (test.txt) creation, `cp test.txt out.txt` is executed.

It may take a little time until filesystem watch starts, so this code (L27~31) confirms the file (out.txt) existence several times just in case.

Reply:
Why not just:
- `fork do` to start the process.
- Wait 10 seconds 
- create input file
- check output

Reply:
I think it also works,
but the correct code can finish earlier because it checks every seconds.

If "first wait N seconds" way is better as formula code, I'll change to it.
What do you think?

Reply:
It's more in line with what we normally have so I'd prefer that option.